.coderabbit.yaml

```
@@ -0,0 +1,25 @@
+    drafts: false
+  path_filters:  # Exclude these paths
+    - "!docs/**"
+    - "!public/**"
+    - "!**/*.md"
+
+chat:
+  auto_reply: true
+
+# Privacy settings (optional)
+privacy:
+  shareCode: false
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)